README.md

```diff
@@ -100,6 +100,7 @@ Supported libraries are listed
 
 Additional recommended Splunk specific instrumentations:
 
+- [`splunksql`](./instrumentation/database/sql/splunksql)
```

Comment:
Can we rename the package to `otelsql` as it doesn't contain any Splunk-specific features?

Reply:
This instrumentation does have opinions about what spans and span attributes are created. All recommendations from OTel are included here to provide the best user experience in Splunk. This is somewhat a Splunk-specific feature.

Additionally, it would seem that we want to remain consistent with the existing package naming scheme (https://github.com/signalfx/splunk-otel-go/tree/main/instrumentation/net/http/splunkhttp). Right?

Reply:
If it contains spans/attributes not included in specs then you are right that it should be `splunksql` 👍 I will take a closer look on Thursday (if life allows).

`splunkhttp` contains ONLY Splunk specific instrumentation and it is meant to be used together with `otelhttp` from contrib.

Reply:
So far I do not see anything that would say that we cannot name it `otelsql`.

The span name is IMO the best we can do. Regarding "which span attributes are created" we can always make it configurable later. The defaults look quite good to me. 👍

Reply:
I would rather keep it as is.

Reply:
@owais @TomRoSystems WDYT?

Reply:
I created https://github.com/signalfx/splunk-otel-go/issues/102 .

I think we can always change the package name later and just stop making releases for the old one. Especially before GA.

Resolving.